src/main/scala/io/iohk/ethereum/metrics/MetricsConfig.scala

```diff
@@ -38,6 +38,16 @@ object MetricsConfig {
     val environment = config.getString(Keys.Environment)
     val deployment = config.getString(Keys.Deployment)
 
+    if(enabled) {
```

Comment:
But if it's disable, then not specifying `environment` or `deployment` will still throw an exception, right?